src/components/structures/MatrixChat.js

```diff
@@ -361,6 +365,7 @@ module.exports = React.createClass({
                         room_id: result.room_id,
                         event_id: payload.event_id,
                         invite_sign_url: payload.invite_sign_url,
+                        room_oob_data: payload.room_oob_data,
```

Comment:
The reason I used the room_prefix here is because to clarify that it pertains to the room, since it's in MatrixChat, where in RoomView and below, they only handle the room so it's obvious. I don't mind though.